packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -46,7 +50,10 @@ export class Meter implements types.Meter {
   createMeasure(
     name: string,
     options?: types.MetricOptions
-  ): Metric<MeasureHandle> {
+  ): types.Metric<types.MeasureHandle> {
+    if (!this._isValidName(name)) {
+      return NOOP_MEASURE_METRIC;
```

Comment:
I think we should `log` a warning about invalid name and falling back on noop implementation.